### DISCUSSION TITLE
Modifications to support all vehicles on an OnStar Account

### DISCUSSION THREAD

**Discussion by almoney** [BigThunderSR/homeassistant-addons-onstar2mqtt#488] [Ideas]
I am looking to change to support all vehicles on an account.

**Comment by BigThunderSR** [comment]
How many vehicles do you have?

**Reply by almoney** [reply]
Sorry I missed this. I have 4.

**Reply by BigThunderSR** [reply]
> Sorry I missed this. I have 4. 

@almoney, I added Vehicle 3 and Vehicle 4 as options, so that should cover your needs.

**Comment by BigThunderSR** [comment]
Closing due to no response. Please re-open if necessary and provide a response to my question. Thanks.
